lib/files.c

```diff
@@ -789,3 +789,92 @@ tool_rc files_tpm2b_attest_to_tpms_attest(TPM2B_ATTEST *quoted, TPMS_ATTEST *att
 
     return tool_rc_success;
 }
+
+tool_rc files_load_unique_data(const char *file_path,
+TPM2B_PUBLIC *public_data) {
+
```

Comment:
```suggestion

 /*
       * TPM2_MAX_RSA_KEY_BYTES which expands to 512 bytes is the maximum unique
       * size for RSA and other key types.
       *
       * Additionally this may still prove to be a bigger value than what has been
       * implemented in a TPM. For example the value of MAX_RSA_KEY_BYTES is 256
       * for ibmSimulator as specified in implementation.h
       */
```